releasenotes-dca/notes/forward-pld-queries-to-leader-67ab5a941c65a47c.yaml

```diff
@@ -0,0 +1,7 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+enhancements:
+  - |
+    Cluster-Agent Followers can now proxy patch request queries to the Leader for Library Injection with language detection.
```

Comment:
I don't think this should be included in the release notes.
It's not a user-facing feature and PLD is not GA yet.